src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -127,6 +127,13 @@ func (r *Cluster) validateTLS() field.ErrorList {
 				r.Spec.Configuration.TLS.RequireClientAuth,
 				"KafkaAPIEnabled has to be set to true for RequireClientAuth to be allowed to be true"))
 	}
+	if r.Spec.Configuration.TLS.IssuerRef != nil && r.Spec.Configuration.TLS.NodeSecretRef != nil {
+		allErrs = append(allErrs,
```

Comment:
Should we be verifying that both name and namespace are provided?

Reply:
I think actually you can omit that, but then I think you're saying your namespace is default? 🤔 I'll check what core types using this are doing

Reply:
yeah I think empty basically means default, as it does if you omit ns anywhere else. Maybe that's fine? 🤔